tasks/eva_2357/populate_ids.py

```diff
@@ -0,0 +1,196 @@
+import hashlib
+import argparse
+import pymongo
+import traceback
+import logging
+from ebi_eva_common_pyutils.config_utils import get_mongo_uri_for_eva_profile
+
+
+def generate_update_statement(hash_to_variant_ids, hash_to_accession_info):
+    variant_to_ids = {}
+    update_statements = []
+    for sve_hash, accessions in hash_to_accession_info.items():
+        ss_accession = f"ss{accessions.get('ss')}"
+        rs_accession = f"rs{accessions.get('rs')}" if accessions.get('rs') else accessions.get('rs')
```

Comment:
Suggest hardcoding else to None...
```suggestion
        rs_accession = f"rs{accessions.get('rs')}" if accessions.get('rs') else None
```